requirements-external.txt

```diff
@@ -7,3 +7,5 @@ git+https://github.com/opendatahub-io/oauthenticator@42c4790a2db77ff4259a74e15ed
 # jupyterhub-singleuser-profiles/jupyterhub_singleuser_profiles/ui/package-lock.json
 #
 git+https://github.com/red-hat-data-services/jupyterhub-singleuser-profiles@d6c088f96a95a92c9aa8aa2a3fe657552ad82071#egg=jupyterhub-singleuser-profiles
+git+https://github.com/opendatahub-io/traefik-proxy@b468da04bf0057e9baeb458de6c3f9f677cfb79e#edd=jupyterhub-traefik-proxy
```

Comment:
typo here, "edd" instead of "egg"

otherwise lgtm

Reply:
ah shoot lol

Reply:
updated